hazelcast-sql/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -58,6 +58,41 @@ SqlCreate SqlCreateMapping(Span span, boolean replace) :
     }
 }
 
+/**
+ * Parses CREATE DATA LINK statement.
+ */
+SqlCreate SqlCreateDataLink(Span span, boolean replace) :
+{
+    SqlParserPos startPos = span.pos();
+    boolean ifNotExists = false;
+    SqlIdentifier name;
+    SqlIdentifier type;
+    SqlNodeList sqlOptions;
+}
+{
+    <DATA> <LINK>
+    [
+        <IF> <NOT> <EXISTS> { ifNotExists = true; }
```

Comment:
no `CREATE OR UPDATE DATA LINK` and `ALTER DATA LINK`? Is the only way to change the datalink to drop it and create it again? How does it affect dependent mappings?

Reply:
ok, `OR REPLACE` variant is supported